src/BloomExe/web/EnhancedImageServer.cs

```diff
@@ -175,17 +174,6 @@ public static SimulatedPageFile MakeRealPublishModeFileInBookFolder(HtmlDom dom)
 			return new SimulatedPageFile() { Key = uri.AbsoluteUri };
 		}
 
-		private static string RemoveLocalhostReferenceFromPath(string path)
-		{
-			if (String.IsNullOrEmpty(path) || !path.StartsWith(PathEndingInSlash))
-				return path;
-			path = path.Substring(PathEndingInSlash.Length - 1);
-			path = Regex.Replace(path, "^/([A-Z])%3A/", "$1:/");
```

Comment:
The line
		path = path.Substring(PathEndingInSlash.Length - 1);
ensures that the path always starts with a slash.  Removing the leading slash is forced by that operation.  The string.ToLocalhost() extension Does The Right Thing here as its replacement.